src/Microsoft.Extensions.Configuration.AzureAppConfiguration/Constants/RefreshConstants.cs

```diff
@@ -8,12 +8,12 @@ namespace Microsoft.Extensions.Configuration.AzureAppConfiguration
     internal class RefreshConstants
     {
         // Key-values
-        public static readonly TimeSpan DefaultCacheExpirationInterval = TimeSpan.FromSeconds(30);
-        public static readonly TimeSpan MinimumCacheExpirationInterval = TimeSpan.FromSeconds(1);
+        public static readonly TimeSpan DefaultRefreshInterval = TimeSpan.FromSeconds(30);
+        public static readonly TimeSpan MinimumRefreshInterval = TimeSpan.FromSeconds(1);
 
         // Feature flags
-        public static readonly TimeSpan DefaultFeatureFlagsCacheExpirationInterval = TimeSpan.FromSeconds(30);
-        public static readonly TimeSpan MinimumFeatureFlagsCacheExpirationInterval = TimeSpan.FromSeconds(1);
+        public static readonly TimeSpan DefaultFeatureFlagsRefreshInterval = TimeSpan.FromSeconds(30);
+        public static readonly TimeSpan MinimumFeatureFlagsRefreshInterval = TimeSpan.FromSeconds(1);
```

Comment:
Can we use `FeatureFlag` (singular) in the name to match other places?

Reply:
I was trying to think why this had an "s" anyway, is it just to show that it applies to all flags?

Reply:
Might've been an oversight earlier. 'Secret' refresh interval is also singular.